### PR TITLE
fix: 🐛 [IOSSDKBUG-485]Filter layout issue for very large fonts

### PR DESCRIPTION
**Pull request by restaurantt**
Use NavigationStack instead of HStack, adapt to dynamic type changing.

### CODE REVIEW

File: Sources/FioriSwiftUICore/Views/OptionListPickerItem+View.swift [hunk 1]
```diff
@@ -22,6 +22,24 @@ public enum OptionListPickerItemLayoutType {
     case flexible
 }
 
+public extension OptionListPickerItem {
+    /// create a filter picker which used in FilterFeedbackBarItem
+    /// - Parameters:
+    ///   - value: Selected value indexs.
```
Comment (dyongxu) [inline Sources/FioriSwiftUICore/Views/OptionListPickerItem+View.swift:28]:
```suggestion
    ///   - value: Selected value index.
```

**Review by dyongxu — COMMENTED** [review]
doc update

File: Sources/FioriSwiftUICore/Views/OptionListPickerItem+View.swift (hunk 1 above)
Comment (restaurantt) [reply Sources/FioriSwiftUICore/Views/OptionListPickerItem+View.swift:28]: value is [Int] type. Should it be plural?

Comment (restaurantt) [reply Sources/FioriSwiftUICore/Views/OptionListPickerItem+View.swift:28]: Change to 'Indexes for selected values.'

**Review by dyongxu — APPROVED** [review]

**Pull request merged**